README.md

```diff
@@ -2,6 +2,9 @@
 Docker image of ROS (Robot Operating System) on Alpine Linux
 
 ## News
+### July 2025
+`noetic-3.17` and `humble-3.17` (Alpine 3.17 EOL on Nov 2022) are dropped. The corresponding images are still available but no longer updated.
```

Comment:
@Taka-Kazu Nov 2022 is the release date. EOL date should be on Nov 2024
https://endoflife.date/alpine-linux